manifest/kubernetes/ubuntu/ncp-ubuntu.yaml

```diff
@@ -949,21 +949,38 @@ spec:
           - name: host-var-run
             mountPath: /var/run/openvswitch
             subPath: openvswitch
+{{if .UseNsxOvsKmod}}
```

Comment:
Hi Shizhao, 
Usually, we sync manifest files from the ones generated from ncp side. Do you think we could make such change in ncp manifest templates so that the generated  manifests are consistent with operator side? 

In addition, we might need to make the same change on other manifestss as well:
manifest/kubernetes/rhel/ncp-rhel.yaml
manifest/openshift4/coreos/ncp-openshift4.yaml

Thanks

Reply:
We have options in manifest templates to control what contents are necessary for operator env or other platform.

Reply:
Hi Yun,

I've made changes on ncp-rhel and ncp-openshift4.yaml

Reply:
Hi Shizhao, 
Thanks for updates on all platforms.